src/actions/walletAction.ts

```diff
@@ -17,8 +17,49 @@ import State, {
   stringToKeyStoreType,
 } from '../state';
 
+// global variables
 const state = new State();
 
+const password = new enquirer.Password({
+  type: 'password',
+  name: 'password',
+  message: 'Type your password',
+});
+
+// wallet: generate a new address
+async function walletAddress() {
```

Comment:
lets' move this in a specific wallet action file, so it becomes a proper subcommand like `balance` and `send`